### ISSUE TITLE
[wiki] "Compiling from Source (Windows)" outdated

### ISSUE THREAD

**Issue by fireandfuel** [clementine-player/Clementine#5656]
I followed the wiki article to test my changes for the Windows build and found out the article is out of date.

Compiling the dependencies needs the following things installed:
- autoconf 2.69 (not in Ubuntu precise repository)
- automake 1.14  (not in Ubuntu precise repository)
- protobuf 2.6.1 (not in Ubuntu precise repository)

The cmake command to configure the Windows build of clementine is wrong, it's `/target/bin/protoc.exe` and not `/target/bin/protoc`.

The correct command is:
```PKG_CONFIG_LIBDIR=/target/lib/pkgconfig \
    cmake .. \
    -DCMAKE_TOOLCHAIN_FILE=/src/Toolchain-mingw32.cmake \
    -DQT_HEADERS_DIR=/target/include \
    -DQT_LIBRARY_DIR=/target/bin \
    -DPROTOBUF_PROTOC_EXECUTABLE=/target/bin/protoc.exe
```

Wine needs to be reconfigured to build  clementine, otherwise it won't find some libraries.
~/.wine/system.reg
```
[System\\CurrentControlSet\\Control\\Session Manager\\Environment]
"PATH"=str(2):"C:\\windows\\system32;C:\\windows;C:\\windows\\system32\\wbem;Z:\\src\\windows\\clementine-deps\\"
```

**Comment by jonaski** [comment]
It uses mingw, not wine. I use mxe instead: http://mxe.cc/

**Comment by fireandfuel** [comment]
@jonaski You are right, mingw is the build environment, but if you have a look at [Compiling form Source (Windows)](https://github.com/clementine-player/Clementine/wiki/Compiling-from-Source-%28Windows%29) wine is a build dependency. But it's still broken and needs to be updated.
If you have a better way to build it, please update the wiki entry.
